Content.Client/Atmos/EntitySystems/EnabledAtmosDeviceVisualizerSystem.cs

```diff
@@ -0,0 +1,21 @@
+using Content.Client.Atmos.Components;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Atmos.EntitySystems;
+
+public abstract class EnabledAtmosDeviceVisualizerSystem<T> : VisualizerSystem<T>
+    where T: EnabledAtmosDeviceComponent
+{
+    protected abstract object LayerMap { get; }
+    protected abstract Enum DataKey { get;  }
+
```

Comment:
As general code design feedback, You should probably have just made these data-fields on the `EnabledAtmosDeviceComponent` and had a single enabled device visualizer system, rather than having lots of systems & components.

Other than that, yeah this is pretty much how the conversion should be done, except for the fact that all these visualizers should probably just use the generic visualizer. I added a comment to ##8995 explaining how to use it, sorry for not doing so earlier.

Reply:
No problem. I'll go give that a read and see what I can do :+1:

Reply:
Converted everything I could to generic visualizers. I've also pushed a PR to the engine to fix an issue with the wrong enum being mapped for the air scrubber.